modules/local/subworkflow/markduplicates.nf

```diff
@@ -4,9 +4,13 @@
 ================================================================================
 */
 
-params.markduplicates_options = [:]
+params.adam_transformalignments_options = [:]
+params.markduplicates_options           = [:]
+params.samtools_index_options           = [:]
 
-include { GATK_MARKDUPLICATES } from '../../nf-core/software/gatk/markduplicates' addParams(options: params.markduplicates_options)
+include { ADAM_TRANSFORMALIGNMENTS } from '../../nf-core/software/adam_transformalignments' addParams(options: params.adam_transformalignments_options)
```

Comment:
It might be preferable to structure this into `software/adam` as with GATK, and use `markduplicates` as the process name.  `transformAlignments` in ADAM is multi-purpose, it also does indel realignment, BQSR, sorting, and other transformations.

Reply:
Good point, I'll fix that